cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -25,18 +25,21 @@
 import javax.annotation.Nullable;
 
 /**
- * Interface providing the HBase DDL operations.
+ * Interface providing the HBase DDL operations. Except the {@link #initialize} method,
```

Comment:
Also would be worthwhile to say why, something like 'must be idempotent in order to allow retry of failed operations'